api/jobs/jobs.py

```diff
@@ -83,10 +125,14 @@ def load(cls, e):
         # Don't modify the map
         d = copy.deepcopy(e)
 
-        if d.get('inputs', None):
-            inputs = d['inputs']
-            for x in inputs.keys():
-                inputs[x] = create_filereference_from_dictionary(inputs[x])
+        if d.get('inputs'):
+            input_dict = {}
+
+            for i in d['inputs']:
```

Comment:
Using this we could have multiple inputs of `input` 'dicom' in one list but it wouldn't be allowed in any other document.